scripts/cmake/MCWAMP.cmake

```diff
@@ -20,7 +20,10 @@ set(MCWAMP_INC_DIR "${PROJECT_SOURCE_DIR}/include")
 
 macro(amp_target name )
 	target_compile_definitions(${name} PRIVATE "GTEST_HAS_TR1_TUPLE=0")
-	target_include_directories(${name} PRIVATE ${GTEST_INC_DIR} ${LIBCXX_INC_DIR} ${MCWAMP_INC_DIR})
+  target_include_directories(${name} SYSTEM PRIVATE ${GTEST_INC_DIR} ${LIBCXX_INC_DIR})
+	target_include_directories(${name} PRIVATE ${MCWAMP_INC_DIR})
+  target_include_directories(${name} SYSTEM INTERFACE $<INSTALL_INTERFACE:$<INSTALL_PREFIX>/include>)
+  target_include_directories(${name} SYSTEM INTERFACE /usr/include/libcxxabi)
```

Comment:
The first two lines make the gtest and libcxx includes(which I don't think this variable is set) system include, however the mcwamp include is not since its part of the project. 

The next line adds include to the directory where it will be installed(but only when its installed and not built). This is so consumers of the target can include the header files. 

The last line adds an include for libc++abi, I recall this was needed at one point. Perhaps not, I will drop it as I don't see any libc++abi header included.